### PR TITLE
Fix btn-block Bootstrap challenge

### LINKED ISSUE #8284: typo in bootstrap block-level btn example
I believe the examples for bootstraps block-level button class is incorrect and may cause confusion. Examples are showing " .btn.btn-block" when they should be "btn btn-block" in order to work 

#### Challenge Name
Create a Block Element Bootstrap Button

https://www.freecodecamp.com/challenges/create-a-block-element-bootstrap-button
#### Browser Information

* Browser Name, Version: Chrome Canary
* Operating System: OS X El Capitan vs 10.11.3
* Mobile, Desktop, or Tablet: Mackbook Pro Laptop

### PR DESCRIPTION
**Pull request by Bouncey**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (e.g. `Closes #XXXX`): Closes #8284 
#### Description

Fixes issue with image for describing the difference between btn and
btn-block. The image was deemed confusing with regards to the Bootstrap
btn classes

Unable to test locally

### CODE REVIEW

**Comment by Bouncey** [comment]
I was unable to test, I have spent a good portion of the day trying to set up a testing environment on my windows laptop, but i can't get it to work. 

I have only edited text inside the JSON for the challenge

**Comment by raisedadead** [comment]
@Bouncey will test this one for you. I am aware that you spent a huge amount of time trying to get this setup running locally.Thanks for your patience and thanks for contributing.

On a side note, if you are looking forward to contribute more, you can have a look at our wiki repo. It does not require any setup other than that you have had already (Git and a Text Editor).

File: seed/challenges/01-front-end-development-certification/bootstrap.json [hunk 1]
```diff
@@ -365,9 +365,7 @@
...
       "description": [
-        "Normally, your <code>button</code> elements are only as wide as the text that they contain. By making them block elements, your button will stretch to fill your page's entire horizontal space and any elements following it will flow onto a \"new line\" below the block.",
-        "This image illustrates the difference between <code>inline</code> elements and <code>block-level</code> elements:",
-        "<a href=\"//i.imgur.com/O32cDWE.png\" data-lightbox=\"img-enlarge\"><img class=\"img-responsive\" src=\"//i.imgur.com/O32cDWE.png\" title=\"Click to enlarge\" alt=\"An \"inline\" button is as small as the text it contains. In this image, it's centered. Below it is a \"block-level\" button, which stretches to fill the entire horizontal space.'></a>",
+        "Normally, your <code>button</code> elements with the class <code>btn</code> are only as wide as the text that they contain. By making them block elements with the additional class of <code>btn-block</code>, your button will stretch to fill your page's entire horizontal space and any elements following it will flow onto a \"new line\" below the block.",
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/bootstrap.json]: While you have removed the image. You would need to add an code snippet in as example in here.

**Comment by raisedadead** [comment]
@Bouncey Please add a code snippet as mentioned in the comments above.
/cc @erictleung

**Comment by raisedadead** [comment]
@Bouncey When you add the code snippet, remember to 

```sh
git commit --amend
git push origin FIX/btn-block-Bootstrap-challenge
```

**Comment by erictleung** [comment]
Maybe this code snippet and text would help underneath the changes.

> Buttons like this will only be as wide as the text. In this example, it'll only be as wide as the text "Press."
> ```
> <button class="btn">Press</button>
> ```
> When you add the class `btn-block`, the button will now take up the page's horizontal space.
> ```
> <button class="btn btn-block">Press</button>
> ```

**Comment by Bouncey** [comment]
Commit amended inline with feedback, passes local tests

**Comment by raisedadead** [comment]
It looks good to me.
@erictleung Please can you have a look and merge as well?

**Comment by erictleung** [comment]
@Bouncey looks great! I like the actual buttons there. Two things:

1.  Can you put the code for the buttons on their own lines?

    image

2. Can you [squash your commits](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Squash) to one?

Otherwise looks great to me 👍

**Comment by erictleung** [comment]
@Bouncey I was thinking that it would be its own line, like so:

```
"<code>&lt;button class=\"btn\"&gt;Submit&lt;/button&gt;</code>",
"This button would only be as wide as the word \"Submit\".",
```

You've almost squashed your commits to one but there are still [three commits](https://github.com/FreeCodeCamp/FreeCodeCamp/pull/8538/commits).

**Comment by Bouncey** [comment]
@erictleung I wasn't too sure what you meant, but sure I can do that. I might just have an issue over how i've edited them though. @raisedadead is having a look for me

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **AWESOME.** :sparkles: :sparkles: 

:tada: Thanks @Bouncey and congratulations for your first submission to Free Code Camp :tada:

**Pull request merged**